.github/workflows/setup-and-test.yml

```diff
@@ -27,6 +27,9 @@ jobs:
       - name: Run tests
         run: CI=true yarn test
 
+      - name: Run linter
```

Comment:
Would it make sense to run linter before running tests?

Reply:
That's a great point, they both have the potential to fail the build but linting should be much faster, resulting in a lower average time to result

Reply:
If possible in parallel would be great (ie get both results at the same time from the same PR, and not have to bounce backwards and forwards between code fixes and lint fixes)

Reply:
The obvious way to run in parallel would be to create a new YAML and run two workflows, I'm not sure the current 5 second lint warrants that.

Ideally developers should never find out about linting problems from CI alone because local tools tell them in real time. (Any dev workflow that requires pushing to a remote to try changes gets a 👎 from me.)

Reply:
I changed my mind. The real benefit from multiple workflows is easily diagnosing whether it was linting or tests that failed, without having to click into the log.